frontend/src/app/app.component.html

```diff
@@ -2,6 +2,12 @@
 <a tabindex="0" class="usa-skipnav" href="#skip-nav-target" (click)="util.gotoHashtag('skip-nav-target', $event)">Skip to main content</a>
 <header id="header" role="banner" class="usa-header usa-header-extended">
   <app-usa-banner></app-usa-banner>
+  <div class="usa-alert usa-alert-warning application-deadline-warning application-browser-alert" aria-role="dialog" aria-label="warning" *ngIf="browserName != 'Chrome'">
```

Comment:
This will work just fine but I'd recommend getting in the habit of using `!==` and `===`.

From https://developer.mozilla.org/en-US/docs/Web/JavaScript/Equality_comparisons_and_sameness:

"In most cases, using loose equality is discouraged. The result of a comparison using strict equality is easier to predict, and may evaluate more quickly due to the lack of type coercion."